site/globalDetailsOnDemand.scss

```diff
@@ -0,0 +1,18 @@
+.global-dod {
```

Comment:
I'm not sure this is a great name - in my thinking, we will use global DoD everywhere and have no other DoD implementation?
`dod-tooltip` would be a better name, then.

Reply:
lol yeah the naming here was extremely "first thing that came to mind" 😛

once the pre-existing dod stuff is stripped out i'll be able to have better names